connector-pagerduty/src/main/java/com/redhat/cloud/notifications/connector/pagerduty/PagerDutyTransformer.java

```diff
@@ -25,23 +25,27 @@ public class PagerDutyTransformer implements Processor {
 
     public static final String ACCOUNT_ID = "account_id";
     public static final String APPLICATION = "application";
+    public static final String APPLICATION_URL = "application_url";
     public static final String BUNDLE = "bundle";
     public static final String CLIENT = "client";
     public static final String CLIENT_URL = "client_url";
     public static final String CONTEXT = "context";
     public static final String CUSTOM_DETAILS = "custom_details";
     public static final String DISPLAY_NAME = "display_name";
-    public static final String ENVIRONMENT_URL = "environment_url";
     public static final String EVENT_ACTION = "event_action";
     public static final String EVENT_TYPE = "event_type";
     public static final String EVENTS = "events";
     public static final String GROUP = "group";
+    public static final String HREF = "href";
+    public static final String INVENTORY_URL = "inventory_url";
+    public static final String LINKS = "links";
```

Comment:
If I understood properly, `LINKS` property will include only one link right? then shouldn't we remove the `s`?